src/examples/eet/eet-file.c

```diff
@@ -32,7 +32,12 @@ create_eet_file(void)
      "\x79\x20\x6c\x6f\x61\x64\x65\x64\x20\x69\x6e\x20\x6d\x65\x6d\x6f"
      "\x72\x79\x2e\x0a\x00";
 
-   ef = eet_open("/tmp/my_file.eet", EET_FILE_MODE_WRITE);
+   char *tmpdir = getenv("TEMP");
```

Comment:
Oh, you are using `getenv` hmmm, what about making it more `EFL` like with `eina_environment_tmp_get()`?
```suggestion
   const char * tmpdir = eina_environment_tmp_get();
```

Reply:
Nice, I didn't know that one, I'll use